capi_image_builder/main.py

```diff
@@ -0,0 +1,6 @@
+# Clone repo
```

Comment:
Is this a placeholder for developing the `main.py` in a future PR?

Reply:
Yes, it's the steps (IMO) we need to write out and add unit tests for